tests/test_smoke.py

```diff
@@ -526,7 +526,7 @@ def test_autostop():
     run_one_test(test)
 
 
-# ---------- Testing Autostopping ----------
+# ---------- Testing Autodowning ----------
 def test_autodown():
```

Comment:
Can we add a `sky launch --down` test? I.e., immediately down. Reason: guessing this will be a popular use case.

Reply:
Good point! After we discussed offline, we decide to make the `--down` alone have the same meaning as `--down -i 1` to make sure the cluster will only be torn down after all jobs completed on the cluster. Otherwise, there will be two concepts for e.g. `--down` and `--down -i 4`, creating additional burden.